### ISSUE TITLE
Automatically select a follow frame when selecting a marker topic

### ISSUE THREAD

**Issue by defunctzombie** [foxglove/studio#2401]
**Description**
When a user selects their first marker in an empty 3d panel (or when there is no follow frame selected), automatically assign a follow frame based on the user selection.

**Comment by defunctzombie** [comment]
Blocked on https://github.com/foxglove/studio/issues/2400

**Comment by jtbandes** [comment]
Duplicate of https://github.com/foxglove/studio/issues/2338

**Comment by jtbandes** [comment]
Why was this blocked on #2400?
